R/register_extendr.R

```diff
@@ -10,16 +10,21 @@
 #' `R/extendr-wrappers.R`. Afterwards, you will have to re-document and then
 #' re-install the package for the wrapper functions to take effect.
 #'
-#' @inheritParams pkgload::load_all
 #' @param path Path from which package root is looked up.
 #' @param quiet Logical indicating whether any progress messages should be
 #'   generated or not.
-#' @param force_wrappers Logical indicating whether to install a minimal wrapper
-#'   file in the cases when generating wrappers by Rust code failed. This might
-#'   be needed when the wrapper file is accidentally lost or corrupted.
+#' @param force Logical indicating whether to force re-generating
+#'   `R/extendr-wrappers.R` even when it doesn't seem to need updated (By
+#'   default, generation is skipped when it's newer than the DLL).
```

Comment:
Just a minor quibble: either use lower case "by" or put a period before the parentheses and a second one inside: "...need to be updated. (By default, ... the DLL.)"